api/src/main/java/io/kroxylicious/testing/kafka/api/KafkaCluster.java

```diff
@@ -60,4 +62,9 @@ public interface KafkaCluster extends AutoCloseable {
      * @return mutable configuration map
      */
     Map<String, Object> getKafkaClientConfiguration(String user, String password);
+
+    /**
+     *
+     */
+    assertEachBrokerDescibesCluster(Predicate<org.apache.kafka.clients.admin.DescribeClusterResult> descibeClusterResultMatcher, Duration timeout);
```

Comment:
Is that actually used?

Reply:
Not yet. 

I need to add a test to verify the behaviour.